### PR TITLE
Added script for checking cert expiration

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1672842

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@gpei PTAL. Thank you!

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -47,6 +47,116 @@ Certificates examined by the role include:
...
 
+To list all OpenShift TLS certificate expiration dates, use the following script:
+
+[IMPORTANT]
+====
+The script execution machine requires password-less SSH access to all nodes.
+====
+
+----
+!/bin/bash
```
Comment (gpei) [inline install_config/redeploying_certificates.adoc:58]: Missing a leading `#` here

**Comment by gpei** [comment]
The script basically work for me when I made a try against a 3.11 cluster.
**But** I still think it's not appropriate to put the script in the official doc:
- This shell script looks discordant in an ansible based section.
- The output of the script is useful but not formal.
```
- secret/heapster-certs -n openshift-infra, field: tls.crt
        Validity
            Not Before: Apr  7 10:47:56 2020 GMT
            Not After : Apr  7 10:47:57 2022 GMT
--
        Validity
            Not Before: Apr  7 08:44:48 2020 GMT
            Not After : Apr  6 08:44:49 2025 GMT
------------------------- all nodes' kubelet TLS certificate -------------------------
qe-smoke311-master-etcd-1 - /etc/origin/node/certificates/kubelet-client-2020-04-07-08-45-57.pem
        Validity
            Not Before: Apr  7 08:44:48 2020 GMT
            Not After : Apr  7 08:44:49 2022 GMT
```
- There's also no enough prerequisites mentioned to run this script, like where to run the script, how to make sure oc works, how to prepare password-less SSH access to all nodes 
- There's many hardcoded secret and project names in the script, it won't work when user changed them
- Especially for some components like metrics, logging which were not installed by default, it will just throw out such error:
```
- secret/metrics-server-certs -n openshift-metrics-server, field: ca.crt
Error from server (NotFound): namespaces "openshift-metrics-server" not found
- secret/metrics-server-certs -n openshift-metrics-server, field: tls.crt
Error from server (NotFound): namespaces "openshift-metrics-server" not found
- secret/logging-elasticsearch -n openshift-logging, field: admin-ca
Error from server (NotFound): namespaces "openshift-logging" not found
- secret/logging-elasticsearch -n openshift-logging, field: admin-cert
Error from server (NotFound): namespaces "openshift-logging" not found
- secret/logging-curator -n openshift-logging, field: ca
Error from server (NotFound): namespaces "openshift-logging" not found
```

In my opinion, we could provide the [solution](https://access.redhat.com/solutions/3930291) as an external reference link in our doc, for the customers who want to check more certificates that our playbook didn't covered. @sdodson wdyt?

**Comment by ahardin-rh** [comment]
@gpei Good point. I updated this to link to the solution instead.

**Pull request merged**